sequencer/src/l1_client.rs

```diff
@@ -129,6 +130,34 @@ impl L1Client {
             }
         }
     }
+    /// Get fee info for each `Deposit` ocurring between `prev`
+    /// and `new`. Returns `Vec<FeeInfo>`
+    pub async fn get_pending_deposits(
```

Comment:
I think "pending" is a bit misleading because pending transactions usually refer to transactions that haven't been included in block so I think `get_finalized_deposits` would be a better choice of name here.

Reply:
:thinking: I take "pending" to refer to sequencer state (as opposed to the l1), which makes sense to me. I don't have a strong opinion but ticket (#1119) *does* say "pending".

Reply:
In some ways they are pending to be credited in our own ledger but since this is a method on the L1 client I do think it's better to refer to them as "finalized deposits" because they are finalized on L1.

Reply:
fixed in 52d20ed22bfd3976bbd7b6bdfe5115c05be99379